src/store/card.js

```diff
@@ -30,8 +30,12 @@ export default {
 		cards: [],
 	},
 	getters: {
-		cardsByStack: state => (id) => {
-			return state.cards.filter((card) => card.stackId === id).sort((a, b) => a.order - b.order)
+		cardsByStack: (state, getters) => (id) => {
+			return state.cards.filter(
+				(card) => card.stackId === id
+				&& (card.title.includes(getters.getSearchQuery) || card.description.includes(getters.getSearchQuery))
```

Comment:
Indentation looks a bit odd here, also it would be nicer if we don't do the include calls at all if no search query is set.